src/Codeception/Codecept.php

```diff
@@ -7,7 +7,7 @@
 
 class Codecept
 {
-    const VERSION = "2.4.4";
+    const VERSION = "2.4.5";
```

Comment:
Is it ok to change the version without tagging it?

Reply:
I should have released it yesterday but I was very tired to do that. 
So I plan to make it right now